src/styles/utilits/_variables.scss

```
@@ -0,0 +1,22 @@
+$day_size: 100px;
```

```suggestion
$day-size: 100px;
```
it's better to name variables with kebab-case